tests/integration/bond_test.py

```diff
@@ -262,3 +262,46 @@ def test_set_bond_mac_address(eth1_up):
         current_state = statelib.show_only(('bond99',))
         bond99_cur_state = current_state[INTERFACES][0]
         assert bond99_cur_state[Interface.MAC] == MAC1.upper()
+
+
+@pytest.fixture
+def bond0():
+    with bond_interface(name='bond99', slaves=[]) as bond_state:
+        yield bond_state
+
+
+def assert_mac_address_behavior(bond0):
```

Comment:
The assertion should be named according to what it asserts. Also it should get the state it should assert as an argument. Why does it have `bond0` as an argument?